app/assets/stylesheets/dm/components/page_component.scss

```diff
@@ -152,6 +152,22 @@
     }
   }
 
+  .usa-img {
+    object-fit: cover;
```

Comment:
I left this in here because the uswds does not seem to have a way to resize images. With this in place an image that has a greater height than width will be cropped vertically when filling out the width, with it removed the image will stretch to fit:

note: image 1's dimensions allow for full size with no cropping, image 2 is taller than wide so it gets cropped when filling the width:
![Screenshot 2023-07-19 at 12 59 27 PM](https://github.com/department-of-veterans-affairs/diffusion-marketplace/assets/65036872/ae5ccdab-8c09-4e1b-943f-c5988c61b548)
e will crop rather than stretch:


With it taken out the image will stretch:
![Screenshot 2023-07-19 at 1 01 48 PM](https://github.com/department-of-veterans-affairs/diffusion-marketplace/assets/65036872/ec9f08c0-45c6-4494-acd1-22af81a00229)

Reply:
I know we are trying to stick to uswds utilities but I think this small bit of customization keeps things looking like the mockup?

Reply:
sounds good 👍 thanks for posting those examples